proxy-node-shared/src/main/java/uk/gov/ida/notification/saml/EidasResponseBuilder.java

```diff
@@ -65,14 +65,18 @@ public EidasResponseBuilder withAssertionSubject(String subject) {
         return this;
     }
 
+    public EidasResponseBuilder addAssertionAttributeStatement(List<Attribute> attributes) {
```

Comment:
It looks like the only place this method is used now is in the new `withLoa` method. Can we just rename it this to `withLoa` rather than calling it from somewhere else?

Reply:
I'm assuming you mean the `addAssertionAuthnStatement` method. I left it that way because the LOA statement just happens to also be an AuthnStatement in the eIDAS SAML spec so I made the LOA call the generic element method, so I think that makes sense.

Reply:
Yeah sorry, that it was I meant. I think it's more complex the way it is. If you were looking at the code you might not have any idea about why it was written that way?

Reply:
It makes sense if you look at the eIDAS spec, which this class should reflect IMO

Reply:
OK, feels like an unnecessary abstraction to me but not a deal breaker.